src/server/server_family.cc

```diff
@@ -382,6 +370,11 @@ void SetMaxClients(std::vector<facade::Listener*>& listeners, uint32_t maxclient
   }
 }
 
+void SetSlowLogMaxLen(util::ProactorPool& pool, uint32_t val) {
```

Comment:
Similarly, introduce `SetSlowlogThreashold`, that will store `log_slower_than_usec` inside slowlog as a variable. See my about not accessing the flag directly. Also introduce the const accessor to this flag.